config/sync/migrate_plus.migration.cs_opportunity_migration.yml

```diff
@@ -0,0 +1,149 @@
+id: cs_opportunity_migration
+class: null
+field_plugin_method: null
+cck_plugin_method: null
+migration_tags:
+  - JSON
+  - Spotlight
+migration_group: custom_content
+label: 'Import JSON'
+source:
+  plugin: url
+  data_fetcher_plugin: http
+  data_parser_plugin: json
+  urls: 'https://cardinalservice-dev.sites-pro.stanford.edu/opportunity-exporter'
+  item_selector: 0
+  fields:
+    -
+      name: title
+      label: Title
+      selector: title
+    -
+      name: body
+      label: body
+      selector: body
+    -
+      name: su_opp_sponsor
+      label: 'Opportunity Sponsor'
+      selector: su_opp_sponsor
+    -
+      name: su_opp_submissions_open
+      label: 'Opportunity Open Submission'
+      selector: su_opp_submissions_open
+    -
+      name: ctaUrl
+      label: 'Call to Action Link'
+      selector: su_opp_solo_url
+    -
+      name: ctaTitle
+      label: 'Call to Action Title'
+      selector: su_opp_solo_url_1
+    -
+      name: su_opp_contact_email
+      label: 'Contact Email'
+      selector: su_opp_contact_email
+    -
+      name: su_opp_contact_name
+      label: 'Contact Name'
+      selector: su_opp_contact_name
+    -
+      name: su_opp_contact_phone
+      label: 'Contact Phone'
+      selector: su_opp_contact_phone
+    -
+      name: su_opp_eligibility
+      label: 'Eligibility'
+      selector: su_opp_eligibility
+    -
+      name: learnMoreUrl
+      label: 'Learn more'
+      selector: su_opp_learn_more
+    -
+      name: learnMoreTitle
+      label: 'Learn more Title'
+      selector: su_opp_learn_more_1
+    -
+      name: su_opp_location
+      label: 'Location'
+      selector: su_opp_location
+    -
+      name: su_opp_summary
+      label: 'Summary'
+      selector: su_opp_summary
+  ids:
+    title:
+      type: string
+process:
+  title: title
+  body/value: body
+  body/format:
+    plugin: default_value
+    default_value: stanford_html
+  su_opp_sponsor:
+    -
+      plugin: skip_on_empty
+      source: su_opp_sponsor
+      method: process
+    -
+      plugin: entity_generate
+      entity_type: taxonomy_term
+      bundle_key: vid
+      bundle: opportunity_sponsor
+      value_key: name
+      ignore_case: true
+  su_opp_open_date: su_opp_submissions_open
+  su_opp_cta_url/uri: ctaUrl
+  su_opp_cta_url/title: ctaTitle
+  su_opp_contact_email: su_opp_contact_email
+  su_opp_contact_name:
+    -
+      plugin: skip_on_empty
+      source: su_opp_contact_name
+      method: process
+    -
+      plugin: get
+      source: su_opp_contact_name
+  su_opp_contact_phone: su_opp_contact_phone
+  su_opp_eligibility/value: su_opp_eligibility
+  su_opp_eligibility/format:
+    plugin: default_value
+    default_value: stanford_minimal_html
+  su_opp_learn_more/uri: learnMoreUrl
+  su_opp_learn_more/title: learnMoreTitle
+  su_opp_tags:
+    -
+      plugin: concat
+      delimiter: '|'
+      source:
+        - su_opp_location
+    -
+      plugin: explode
+      delimiter: '|'
```

Comment:
Why are you concatting to just explode again?

also check for empty before any of these steps.

Reply:
@pookmish This has been problematic because I can the location to import but none of the other taxonomy terms for filter. But anyway, I've kind of thrown everything on the wall to see what sticks. That's why this is haywire.

Reply:
@pookmish This has been problematic because I can import the location to import but none of the other taxonomy terms for filter. But anyway, I've kind of thrown everything on the wall to see what sticks. That's why this is haywire.